examples/python/wordcloud/readme.md

```diff
@@ -1,3 +1,49 @@
-# stylecloud
+# wordcloud
 
-TODO
+> Generate stylish wordclouds from any webpage.
+
+<a href="https://wordcloud.saasify.sh">
+  <img
+    src="https://raw.githubusercontent.com/saasify-sh/saasify/master/examples/python/wordcloud/examples/promo.png"
+    alt="Wordcloud Examples"
+  />
+</a>
```

Comment:
This thing isn't deployed yet

Reply:
yep; there's an issue blocking the now deployment from working at the moment that I need to debug. something to do w/ the `Pipfile`